<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">

        <script async="async" src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>

        <!--Swap our for actual link-->
        <link rel="canonical" href="https://datadiversitylab.github.io/publications/">

        <meta name="description" content="Publications of the Data Diversity Lab">
        <meta name="keywords" content="Data Diversity Lab and Development">
        <meta name="google-site-verification" content="u82fOqFBdQHRSP52a0zYaa0TN4pXvDCaFbvtkQINz2M" />

        <!--Social Media Display-->
        <meta property="og:title" content="Publications"/>
        <meta property="og:description" content="Publications of the Data Diversity Lab"/>
        <meta property="og:type" content="website"/>
        <meta property="og:url" content="https://datadiversitylab.github.io/publications/"/>
        <meta property="og:image" content="/images/social.jpg"/>
        <meta property="og:image:secure_url" content="/images/social.jpg"/>

        <!--Favicons-->
        <link rel="apple-touch-icon" sizes="180x180" href="/assets/favicons/apple-touch-icon.png">
        <link rel="icon" type="image/png" sizes="32x32" href="/assets/favicons/favicon-32x32.png">
        <link rel="icon" type="image/png" sizes="16x16" href="/assets/favicons/favicon-16x16.png">
        <link rel="manifest" href="/assets/favicons/site.webmanifest">
        <meta name="msapplication-TileColor" content="#da532c">
        <meta name="theme-color" content="#ffffff">

        <!-- Preload -->
        <link rel="preload" as="image" href="/assets/images/logos/logo-dark.svg">
        <link rel="preload" as="font" type="font/woff2" href="/assets/fonts/roboto-v29-latin-regular.woff2" crossorigin>
        <link rel="preload" as="font" type="font/woff2" href="/assets/fonts/roboto-v29-latin-700.woff2" crossorigin>

        <link rel="stylesheet" href="/css/root.css">
        <link rel="stylesheet" href="/css/dark.css">

        

        <!-- Loading in page specific CSS sheets from css block -->
        <link rel="stylesheet" href="/css/critical.css"/>
        <link rel="stylesheet" href="/css/publications.css">

        <script defer src="https://umami.datadiversitylab.synology.me/script.js" data-website-id="543cffef-a100-47af-9dd9-2cf39517077b"></script>

        <!-- For home page, use full title, including location for SEO. Other pages should just include the page name -->
        <title>
            
                Publications | Data Diversity Lab
            
        </title>
    </head>
    <body>

        <a class="skip" aria-label="skip to main content" href="#main">Click To Skip To Main Content</a>

        <!-- ============================================ -->
<!--                 Navigation                   -->
<!-- ============================================ -->


<header id="cs-navigation">
    <div aria-hidden="true" class="background-color-div">
    </div>
    <div class="cs-container">
        <!--Nav Logo-->
        <a href="/" class="cs-logo" aria-label="back to home">
            
                
                    
                        <img class="cs-dark" src="../../assets/images/logos/ddl-logo.png" alt="logo" width="210" height="29" aria-hidden="true" decoding="async">
                    
                
                    
                
            
        </a>
        <!--Navigation List-->
        <nav class="cs-nav" role="navigation">
            <!--Mobile Nav Toggle-->
            <button class="cs-toggle" aria-label="mobile menu toggle">
                <div class="cs-box" aria-hidden="true">
                    <span class="cs-line cs-line1" aria-hidden="true"></span>
                    <span class="cs-line cs-line2" aria-hidden="true"></span>
                    <span class="cs-line cs-line3" aria-hidden="true"></span>
                </div>
            </button>
            <!-- We need a wrapper div so we can set a fixed height on the cs-ul in case the nav list gets too long from too many dropdowns being opened and needs to have an overflow scroll. This wrapper acts as the background so it can go the full height of the screen and not cut off any overflowing nav items while the cs-ul stops short of the bottom of the screen, which keeps all nav items in view no matter how mnay there are-->
            <div class="cs-ul-wrapper">
                <ul id="cs-expanded" class="cs-ul" aria-expanded="false">
                            <li class="cs-li" tabindex="0">

                                <a  class="cs-li-link"  href="/">
                                    Home
                                </a>
                            </li>
                        
                            <li class="cs-li cs-dropdown" tabindex="0">
                                <span  class="cs-li-link"  href="/the-lab/">
                                    The lab
                                    
                                        <img class="cs-drop-icon" src="../assets/icons/chevron-down.svg" alt="dropdown icon" width="30" height="30" decoding="async" aria-hidden="true">
                                    
                                </span>
                                <ul class="cs-drop-ul">
                                    <li class="cs-drop-li">
                                        <a href="/the-lab/" class="cs-li-link cs-drop-link">The lab</a>
                                    </li>
                                        <li class="cs-drop-li">
                                            <a href="/our-team/" class="cs-li-link cs-drop-link">Our team</a>
                                        </li>
                                        <li class="cs-drop-li">
                                            <a href="/lab-documents/" class="cs-li-link cs-drop-link">Lab documents</a>
                                        </li>
                                        <li class="cs-drop-li">
                                            <a href="/space-and-tech/" class="cs-li-link cs-drop-link">Space and tech</a>
                                        </li>
                                        <li class="cs-drop-li">
                                            <a href="/social/" class="cs-li-link cs-drop-link">Social</a>
                                        </li>
                                        <li class="cs-drop-li">
                                            <a href="/collaborators/" class="cs-li-link cs-drop-link">Collaborators</a>
                                        </li>
                                        <li class="cs-drop-li">
                                            <a href="/join-us/" class="cs-li-link cs-drop-link">Join us</a>
                                        </li>
                                </ul>
                        </li>
                        
                            <li class="cs-li cs-dropdown" tabindex="0">
                                <span  class="cs-li-link"  href="/research/">
                                    Research
                                    
                                        <img class="cs-drop-icon" src="../assets/icons/chevron-down.svg" alt="dropdown icon" width="30" height="30" decoding="async" aria-hidden="true">
                                    
                                </span>
                                <ul class="cs-drop-ul">
                                    <li class="cs-drop-li">
                                        <a href="/research/" class="cs-li-link cs-drop-link">Research</a>
                                    </li>
                                        <li class="cs-drop-li">
                                            <a href="/publications/" class="cs-li-link cs-drop-link">Publications</a>
                                        </li>
                                        <li class="cs-drop-li">
                                            <a href="/grants/" class="cs-li-link cs-drop-link">Grants</a>
                                        </li>
                                        <li class="cs-drop-li">
                                            <a href="/conferences/" class="cs-li-link cs-drop-link">Conferences</a>
                                        </li>
                                        <li class="cs-drop-li">
                                            <a href="/lab-ops/" class="cs-li-link cs-drop-link">LabOps</a>
                                        </li>
                                </ul>
                        </li>
                        
                            <li class="cs-li cs-dropdown" tabindex="0">
                                <span  class="cs-li-link"  href="/teaching/">
                                    Teaching
                                    
                                        <img class="cs-drop-icon" src="../assets/icons/chevron-down.svg" alt="dropdown icon" width="30" height="30" decoding="async" aria-hidden="true">
                                    
                                </span>
                                <ul class="cs-drop-ul">
                                    <li class="cs-drop-li">
                                        <a href="/teaching/" class="cs-li-link cs-drop-link">Teaching</a>
                                    </li>
                                        <li class="cs-drop-li">
                                            <a href="/the-data-hub/" class="cs-li-link cs-drop-link">The Data Hub</a>
                                        </li>
                                        <li class="cs-drop-li">
                                            <a href="/ms-advising/" class="cs-li-link cs-drop-link">MS advising</a>
                                        </li>
                                </ul>
                        </li>
                        
                            <li class="cs-li" tabindex="0">

                                <a  class="cs-li-link"  href="/software/">
                                    Software
                                </a>
                            </li>
                        
                            <li class="cs-li" tabindex="0">

                                <a  class="cs-li-link"  href="/news/">
                                    News
                                </a>
                            </li>
                        
                            <li class="cs-li" tabindex="0">

                                <a  class="cs-li-link"  href="/transparency/">
                                    Transparency
                                </a>
                            </li>
                        
                            <li class="cs-li" tabindex="0">

                                <a  class="cs-li-link"  href="/contact/">
                                    Contact
                                </a>
                            </li>
                        
                </ul>
            </div>
        </nav>
    </div>
</header>
        <main id="main"><!-- ============================================ -->
<!--                  Content                     -->
<!-- ============================================ -->
<script id="altmetric-embed-js" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/altmetric_badges-e7ba41a192efc3e63f5d48f8a1916dec4b1b49c01075bc7b39481408957b3b28.js"></script>
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>

<section id="content-1450" class="cs-top-section">
    <div class="cs-container">
        <div class="cs-content">
            <h1 class="cs-title">Publications</h1>
            <p class="cs-text">
                Browse through our lab's publications and stay informed about our latest findings.
            </p>
        </div>
    </div>
</section>

<div aria-hidden="true" class="line"></div>

<!-- ============================================ -->
<!--                Publications                      -->
<!-- ============================================ -->




<section id="publications-section">
    <div class="cs-container">
        <div class="cs-content cs-sidebar cs-filter-searchable">
            <div class="cs-search-title-wrapper">
                <h6 class="cs-title">Search</h6>
                <div aria-hidden="true" class="line line-full"></div>
                <div class="cs-search-wrapper">
                    <input type="text" id="search" class="cs-search" placeholder="Title or Author">
                    <img src="../assets/icons/order-desc.svg" class="order-by-button" alt="order by" width="25" height="25">
                </div>
                <div class="cs-pi-only-checkbox">
                    <input id="piOnlyCheckbox" type="checkbox"><label for="piOnlyCheckbox">PI only</label>
                </div>
            </div>
            <div class="cs-filter-button-wrapper">
                <div class="cs-add-filter-wrapper">
                    <h6 class="cs-title">Add filters</h6>
                    <img src="../assets/icons/plus.svg" class="plus-button" alt="filter" width="25" height="25">
                </div>

                <div class="cs-filter-year-wrapper">
                    <h6 class="cs-title">Year</h6>
                    <div aria-hidden="true" class="line line-full"></div>
                    <div class="cs-start-end-wrapper">
                        <input type="text" id="startYear" placeholder="Start Year">
                        <input type="text" id="endYear" placeholder="End Year">
                    </div>
                </div>

                <div class="cs-filter-keywords-wrapper">
                    <h6 class="cs-title">Keywords</h6>
                    <div aria-hidden="true" class="line line-full"></div>
                    <div class="cs-filter-wrapper cs-text">
                        <div data-filter="all" class="active filter-option">All (46)</div>
                        
                            
                        
                            
                                <div data-filter="Data science" class="filter-option">Data science (14)</div>
                            
                        
                            
                                <div data-filter="Geosciences" class="filter-option">Geosciences (3)</div>
                            
                        
                            
                                <div data-filter="Paleoclimatology" class="filter-option">Paleoclimatology (1)</div>
                            
                        
                            
                                <div data-filter="Taxonomy" class="filter-option">Taxonomy (14)</div>
                            
                        
                            
                                <div data-filter="Ecology" class="filter-option">Ecology (13)</div>
                            
                        
                            
                                <div data-filter="Software" class="filter-option">Software (5)</div>
                            
                        
                            
                                <div data-filter="Remote sensing" class="filter-option">Remote sensing (2)</div>
                            
                        
                            
                                <div data-filter="Evolution" class="filter-option">Evolution (17)</div>
                            
                        
                            
                                <div data-filter="Climate change" class="filter-option">Climate change (1)</div>
                            
                        
                            
                                <div data-filter="Academia" class="filter-option">Academia (1)</div>
                            
                        
                            
                                <div data-filter="Climatology" class="filter-option">Climatology (1)</div>
                            
                        
                    </div>
                </div>

                <div class="cs-filter-journal-type-wrapper">
                    <h6 class="cs-title">Journals</h6>
                    <div aria-hidden="true" class="line line-full"></div>
                    <div class="cs-filter-wrapper cs-text">
                        <div data-filter="all" class="active filter-option">All (46)</div>
                        
                        
                            <div data-filter="arXiv" class="filter-option">arXiv (1)</div>
                        
                            <div data-filter="Biological Reviews" class="filter-option">Biological Reviews (1)</div>
                        
                            <div data-filter="bioRxiv" class="filter-option">bioRxiv (1)</div>
                        
                            <div data-filter="Biota Colombiana" class="filter-option">Biota Colombiana (1)</div>
                        
                            <div data-filter="BMC Biology" class="filter-option">BMC Biology (1)</div>
                        
                            <div data-filter="Boletin Cientifico Centro de Museos Museo de Historia Natural de la Universidad de Caldas" class="filter-option">Boletin Cientifico Centro de Museos Museo de Historia Natural de la Universidad de Caldas (1)</div>
                        
                            <div data-filter="Boletin del Museo de Entomologia de la Universidad del Valle" class="filter-option">Boletin del Museo de Entomologia de la Universidad del Valle (1)</div>
                        
                            <div data-filter="Caldasia" class="filter-option">Caldasia (1)</div>
                        
                            <div data-filter="Computational Statistics" class="filter-option">Computational Statistics (1)</div>
                        
                            <div data-filter="Comunicaciones en Estadistica" class="filter-option">Comunicaciones en Estadistica (1)</div>
                        
                            <div data-filter="Dugesiana" class="filter-option">Dugesiana (1)</div>
                        
                            <div data-filter="Ecology and Evolution" class="filter-option">Ecology and Evolution (1)</div>
                        
                            <div data-filter="Ecology letters" class="filter-option">Ecology letters (1)</div>
                        
                            <div data-filter="ESS Open Archive" class="filter-option">ESS Open Archive (1)</div>
                        
                            <div data-filter="European Journal of Taxonomy" class="filter-option">European Journal of Taxonomy (1)</div>
                        
                            <div data-filter="Evolution" class="filter-option">Evolution (2)</div>
                        
                            <div data-filter="Evolution letters" class="filter-option">Evolution letters (1)</div>
                        
                            <div data-filter="Frontiers" class="filter-option">Frontiers (1)</div>
                        
                            <div data-filter="Frontiers in Earth Science" class="filter-option">Frontiers in Earth Science (1)</div>
                        
                            <div data-filter="Global Ecology and Biogeography" class="filter-option">Global Ecology and Biogeography (1)</div>
                        
                            <div data-filter="Insect Systematics and Evolution" class="filter-option">Insect Systematics and Evolution (2)</div>
                        
                            <div data-filter="ISPRS Annals of the Photogrammetry, Remote Sensing and Spatial Information Sciences" class="filter-option">ISPRS Annals of the Photogrammetry, Remote Sensing and Spatial Information Sciences (1)</div>
                        
                            <div data-filter="Journal of Biogeography" class="filter-option">Journal of Biogeography (1)</div>
                        
                            <div data-filter="Journal of Evolutionary Biology" class="filter-option">Journal of Evolutionary Biology (1)</div>
                        
                            <div data-filter="Methods in Ecology and Evolution" class="filter-option">Methods in Ecology and Evolution (1)</div>
                        
                            <div data-filter="Molecular Phylogenetics and Evolution" class="filter-option">Molecular Phylogenetics and Evolution (1)</div>
                        
                            <div data-filter="Nature" class="filter-option">Nature (1)</div>
                        
                            <div data-filter="PeerJ" class="filter-option">PeerJ (1)</div>
                        
                            <div data-filter="PNAS" class="filter-option">PNAS (1)</div>
                        
                            <div data-filter="Proceedings of the Royal Society B" class="filter-option">Proceedings of the Royal Society B (1)</div>
                        
                            <div data-filter="Remote Sensing of Environment" class="filter-option">Remote Sensing of Environment (1)</div>
                        
                            <div data-filter="Revista de Biologia Tropical" class="filter-option">Revista de Biologia Tropical (1)</div>
                        
                            <div data-filter="Revista Mexicana de Biodiversidad" class="filter-option">Revista Mexicana de Biodiversidad (1)</div>
                        
                            <div data-filter="Revista MVZ Cordoba" class="filter-option">Revista MVZ Cordoba (1)</div>
                        
                            <div data-filter="The Depositional Record" class="filter-option">The Depositional Record (1)</div>
                        
                            <div data-filter="Zootaxa" class="filter-option">Zootaxa (9)</div>
                        
                    </div>
                </div>
            </div>
        </div>
        <div class="cs-title-card-wrapper">
            <div class="cs-card-group"><div class="cs-item"><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/barmpy.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/barmpy.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/barmpy.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Computational Statistics</span>
                            <span class="cs-name"><p>BARMPy: Bayesian additive regression models Python package</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Danielle Van Boxel</span></span>
                            <span><span class="cs-publication-month">Aug</span>
                                    <span> </span><span class="cs-publication-year">2024</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>We make Bayesian additive regression networks (BARN) available as a Python package, barmpy, with documentation at <a href="https://dvbuntu.github.io/barmpy/">https://dvbuntu.github.io/barmpy/</a> for general machine learning practitioners. Our object-oriented design is compatible with SciKit-Learn, allowing usage of their tools like cross-validation. To ease learning to use barmpy, we produce a companion tutorial that expands on reference information in the documentation. Any interested user can pip install barmpy from the official PyPi repository. barmpy also serves as a baseline Python library for generic Bayesian additive regression models.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Data science</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/vanboxel_2024.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a><a href="https://link.springer.com/article/10.1007/s00180-024-01535-9" target="_blank"><img src="../assets/icons/link.svg" alt="webLink" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/barn.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/barn.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/barn.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">arXiv</span>
                            <span class="cs-name"><p>Bayesian Additive Regression Networks</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Danielle Van Boxel</span></span>
                            <span><span class="cs-publication-month">Mar</span>
                                    <span> </span><span class="cs-publication-year">2024</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>We apply Bayesian Additive Regression Tree (BART) principles to training an ensemble of small neural networks for regression tasks. Using Markov Chain Monte Carlo, we sample from the posterior distribution of neural networks that have a single hidden layer. To create an ensemble of these, we apply Gibbs sampling to update each network against the residual target value (i.e. subtracting the effect of the other networks). We demonstrate the effectiveness of this technique on several benchmark regression problems, comparing it to equivalent shallow neural networks, BART, and ordinary least squares. Our Bayesian Additive Regression Networks (BARN) provide more consistent and often more accurate results. On test data benchmarks, BARN averaged between 5 to 20 percent lower root mean square error. This error performance does come at the cost, however, of greater computation time. BARN sometimes takes on the order of a minute where competing methods take a second or less. But, BARN without cross-validated hyperparameter tuning takes about the same amount of computation time as tuned other methods. Yet BARN is still typically more accurate.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Data science</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2404.04425.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a><a href="https://arxiv.org/abs/2404.04425" target="_blank"><img src="../assets/icons/link.svg" alt="webLink" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/screenshot-2025-01-23-at-8.54.36-am.png">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/screenshot-2025-01-23-at-8.54.36-am.png">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/screenshot-2025-01-23-at-8.54.36-am.png" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">ESS Open Archive</span>
                            <span class="cs-name"><p>Clumped isotope thermometry in foraminifera as a tool in paleoceanography: New planktic and benthic data and constraints on non-thermal effects</p>
</span>
                            <span class="cs-publication-author">Aradhna Tripati, Hannah Tylee Tandy, A Villa, Randy Flores, Alexandrea Jay Arnold, Hannah M Carroll, Maxence Guillermic, Ingrid Maradiaga, Christian Blair, Bisrat Zerehaimanot, Dana Brown, Robert N Ulrich, <span class="cs-highlight-author">Cristian Román-Palacios</span>, Julia de la Cruz, Mohan Kuppusamy, Dharmendra Singh, Raquel Bryant, Fengming Chang, Rob Eagle, Ceara Tomaiso, Thomas M. Marchitto, Rosemarie Came, Jean Lynch-Stieglitz</span>
                            <span><span class="cs-publication-month">Jan</span>
                                    <span> </span><span class="cs-publication-year">2025</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>The carbonate ”clumped” isotope thermometer (Δ47) in foraminifera is increasingly being used to reconstruct ocean temperature. Here we address several less understood aspects of the proxy using a large dataset comprising new and reprocessed data. The Δ47-temperature relationship in foraminifera (n = 260) is described by Δ47 = 0.0374 ± 0.0013 106/T2 + 0.1744 ±0.0154, and in inorganic calcite (n = 118) by Δ47 = 0.0392 ± 0.0014 106/T2 + 0.1547 ±0.0165. Dataset-related differences explain only 11% of the variance; non-thermal effects explain up to 13% of the variance. We address the paucity of benthic data, establishing with more certainty that temperature sensitivity is indistinguishable from planktics and inorganic calcite. The large benthic dataset resolves a previously uncharacterized correlation with [CO32-] that is small (ΔΔ47/ΔCO32- slope = 0.00019 {plus minus}0.00004, n = 66; R2 = 0.315, p &lt;0.01). We report a multivariate regression to account for both temperature and Δ[CO32] for all benthics (epifaunal and infaunal), with Δ47 = 0.152 ± 0.049 + 0.03865 ±0.00376 x 106/T2 + 0.000150 ±0.0000601Δ[CO32-]. We apply these regressions to published Cenozoic benthic Δ47 data, with the multivariate benthic equation yielding temperature and δ18Osw values more consistent with independent proxies, models, and the broader understanding of ocean and cryosphere dynamics under different conditions, including across the Eocene-Oligocene Transition and the Early Eocene Climatic Optimum. In total, this work enables the application of clumped isotopes to foraminifera with a more realistic understanding of uncertainties.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Geosciences</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/1021433_0_merged_1737233943.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a><a href="https://essopenarchive.org/users/529500/articles/1262022-clumped-isotope-thermometry-in-foraminifera-as-a-tool-in-paleoceanography-new-planktic-and-benthic-data-and-constraints-on-non-thermal-effects" target="_blank"><img src="../assets/icons/link.svg" alt="webLink" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/dep2312-fig-0001-m.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/dep2312-fig-0001-m.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/dep2312-fig-0001-m.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">The Depositional Record</span>
                            <span class="cs-name"><p>Comparative clumped isotope temperature relationships in freshwater carbonates</p>
</span>
                            <span class="cs-publication-author">Alexandrea Arnold, John Mering, Lauren Santi, <span class="cs-highlight-author">Cristian Román-Palacios</span>, Huashu Li, Victoria Petryshyn, Bryce Mitsunaga, Ben Elliott, John Wilson, Jamie Lucarelli, Ronny Boch, Daniel Ibarra, Lin Li, Majie Fan, Darrell Kaufman, Andrew Cohen, Rob Dunbar, James Russell, Stefan Lalonde, Priyadarsi D. Roy, Martin Dietzel, Xingqi Liu, Fengming Chang, Robert A. Eagle, Aradhna Tripati</span>
                            <span><span class="cs-publication-month">Oct</span>
                                    <span> </span><span class="cs-publication-year">2024</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Lacustrine, riverine and spring carbonates represent archives of terrestrial climates and their geochemistry has been used to study palaeoenvironments. Clumped isotope thermometry is an emerging tool that has been applied to freshwater carbonates. Limited work has been done to evaluate comparative relationships between clumped isotopes and temperature in different types of modern freshwater carbonates. This study assembles an extensive calibration data set with 135 samples of modern freshwater carbonates from 96 sites and constrains the relationship between independent observations of water temperature and the clumped isotopic composition of carbonates (denoted by Δ47), including new measurements, and recalculates published data in accordance with current community-defined standard values. For temperature reconstruction, the study reports a composite freshwater calibration and material-specific calibrations for biogenic carbonates (freshwater gastropods and bivalves), fine-grained carbonate (e.g. micrites), biologically mediated carbonates (microbialites and tufas) and travertines. Material-specific calibration trends show a convergence of slopes that are in agreement with recently published syntheses, but statistically significant differences in intercepts occur between some materials (e.g. some biogenics, fine-grained carbonates). These differences may arise due to unresolved seasonal biases, kinetic isotope effects and/or varying degrees of biological influence. The impact of different calibrations is shown through application to new data for glacial and deglacial age travertines from Austria and published data sets. While material-specific calibrations may yield more accurate results for biogenic and fine-grained carbonate samples, the use of material-specific and the composite freshwater calibrations generally produces values within 1.0–1.5°C of each other, and typically fall within calibration uncertainty given limitations of precision.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Data science</div><div class="cs-filter">Geosciences</div><div class="cs-filter">Paleoclimatology</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/arnoldetal_2024.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/dep2.312" target="_blank"><img src="../assets/icons/link.svg" alt="webLink" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/large.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/large.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/large.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Insect Systematics and Evolution</span>
                            <span class="cs-name"><p>Holarctic Lineages Cannot Inform Diversity and Evolution in the Neotropics – the barklice family Psocidae as a case study</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Valentina Sarria-Rodríguez</span>, Ranulfo Gonzalez-Obando, Nelson Rivera-Franco, Heiber Cardenas-Henao, <span class="cs-highlight-author">and Cristian Román-Palacios</span></span>
                            <span><span class="cs-publication-month">Apr</span>
                                    <span> </span><span class="cs-publication-year">2024</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Despite tropical species comprising nearly 60% of Psocidae species, previous studies examining the Psocidae phylogeny have undersampled tropical diversity (&lt;40% species in trees). Here we discuss the systematics of the Psocidae based on the most comprehensive species-level sampling of the Psocidae. We infer the phylogenetic position of 43 previously unsampled Neotropical species in the Psocidae phylogeny. We find that Neotropical psocids are generally not closely related to morphologically similar taxa in the Holarctic region. Consequently, the monophyletic status for the major groups within Psocidae (subfamilies and tribes) is recovered only when Holarctic groups are sampled (7–10 of 11 higher-level groups are monophyletic) but violated when Neotropical species are also sampled in the tree (1 of 11 higher-level groups are monophyletic). Our study pinpoints at the downfalls of simply extending taxonomic knowledge from lineages of a certain area (i.e. Holarctic) to inform diversity and evolution of lineages in other regions (i.e. Neotropics).</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/ise_1148_rodriguez_rev5.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a><a href="https://brill.com/view/journals/ise/aop/article-10.1163-1876312X-bja10055/article-10.1163-1876312X-bja10055.xml" target="_blank"><img src="../assets/icons/link.svg" alt="webLink" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/onlinetoolkits.png">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/onlinetoolkits.png">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/onlinetoolkits.png" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Ecology and Evolution</span>
                            <span class="cs-name"><p>Online toolkits for collaborative and inclusive global research in urban evolutionary ecology</p>
</span>
                            <span class="cs-publication-author">Amy Savage, Meredith Willmott, Pablo Moreno, Daijiang Li, Anna Malesis, Zuzanna Jagiełło, Lindsay Miles, Hanna Kuzyo, <span class="cs-highlight-author">Cristian Román-Palacios</span>, David Salazar, Brian Verrelli, Kristin Winchell, Marina Alberti, Elizabeth Carlen, Cleo Falvey, Lauren Johnson, Ella Martin, John Marzluff, Jason Munshi-South, Megan Phifer-Rixey, Ignacy Stadnicki, Marta Szulkin, Santago Bonilla-Bedoya, Yuyu Zhou, Kiyoko Gotanda</span>
                            <span><span class="cs-publication-month">Jun</span>
                                    <span> </span><span class="cs-publication-year">2024</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Urban evolutionary ecology is inherently interdisciplinary. Moreover, it is a field with global significance. However, bringing researchers and resources together across fields and countries is challenging. Therefore, an online collaborative research hub, where common methods and best practices are shared among scientists from diverse geographic, ethnic, and career backgrounds would make research focused on urban evolutionary ecology more inclusive. Here, we describe a freely available online research hub for toolkits that facilitate global research in urban evolutionary ecology. We provide rationales and descriptions of toolkits for: (1) decolonizing urban evolutionary ecology; (2) identifying and fostering international collaborative partnerships; (3) common methods and freely-available datasets for trait mapping across cities; (4) common methods and freely-available datasets for cross-city evolutionary ecology experiments; and (5) best practices and freely available resources for public outreach and communication of research findings in urban evolutionary ecology. We outline how the toolkits can be accessed, archived, and modified over time in order to sustain long-term global research that will advance our understanding of urban evolutionary ecology.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Data science</div><div class="cs-filter">Ecology</div><div class="cs-filter">Software</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/savageetal_2024.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a><a href="https://onlinelibrary.wiley.com/doi/10.1002/ece3.11633" target="_blank"><img src="../assets/icons/link.svg" alt="webLink" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/screenshot-2024-06-15-at-3.35.20-pm.png">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/screenshot-2024-06-15-at-3.35.20-pm.png">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/screenshot-2024-06-15-at-3.35.20-pm.png" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">ISPRS Annals of the Photogrammetry, Remote Sensing and Spatial Information Sciences</span>
                            <span class="cs-name"><p>Open-source automatic extraction of Urban Green Space: Application to assessing improvement in green space access</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Ian Estacio</span>, <span class="cs-highlight-author">Cristian Román-Palacios</span>, Joseph Hoover, Xiaojiang Li, Chris Lim</span>
                            <span><span class="cs-publication-month">Jun</span>
                                    <span> </span><span class="cs-publication-year">2024</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Urban Green Space (UGS) is vital for improving the public health and sustainability of cities. Vector data on UGS such as open data from governments and OpenStreetMap are available for retrieval by interested users, but the availability of UGS data is still limited on global and temporal scales. This study develops the UGS Extractor, a web-based application for the automatic extraction of UGS given user inputs of Area of Interest and Date of Interest. To accommodate various types of green spaces, such as parks or lawns, the application additionally allows users to set parameters for the minimum size of each UGS and the Minimum Urban Neighbor Density, enabling customization of what qualifies as UGS. The UGS Extractor implements a methodological framework that applies object-based image processing, edge detection and extraction, and image neighborhood analysis on the near real-time 10m Dynamic World collection of Land Use/Land Cover images. The application’s utility was demonstrated through two case studies. In the first, the UGS Extractor accurately mapped major parks when compared to open data sources in New Orleans, USA. In the second, the UGS Extractor demonstrated significant increases in the total area of UGS from 2015 to 2023 in Songdo, South Korea, which consequently improved green space accessibility. These results underscore the UGS Extractor’s utility in extracting specific types of UGS and analyzing their temporal trends. This user-friendly application overall offers higher spatial resolution compared to publicly available satellite-based methods while facilitating temporal studies not possible with vector datasets.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Data science</div><div class="cs-filter">Software</div><div class="cs-filter">Remote sensing</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/estacio_2024.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a><a href="https://isprs-annals.copernicus.org/articles/X-2-2024/65/2024/" target="_blank"><img src="../assets/icons/link.svg" alt="webLink" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/samedatadifferentanalyst.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/samedatadifferentanalyst.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/samedatadifferentanalyst.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">BMC Biology</span>
                            <span class="cs-name"><p>Same data, different analysts: variation in effect sizes due to analytical decisions in ecology and evolutionary biology</p>
</span>
                            <span class="cs-publication-author">Elliot Gould, ... [184 authors] ..., <span class="cs-highlight-author">Cristian Román-Palacios</span>, ... [68 authors]..., Emily M. Tompkins</span>
                            <span><span class="cs-publication-month">Feb</span>
                                    <span> </span><span class="cs-publication-year">2025</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Although
variation in effect sizes and predicted values among studies of similar phenomena is inevitable, such variation far exceeds what might be produced by sampling error alone. One possible explanation for variation among results is differences among researchers in the decisions they make regarding statistical analyses. A growing array of studies has explored this analytical variability in different fields and has found substantial variability among results despite analysts having the same data and research question. Many of these studies have been in the social sciences, but one small “many analyst” study found similar variability in ecology. We expanded the scope of this prior work by implementing a large-scale empirical exploration of the variation in
effect sizes and model predictions generated by the analytical decisions of different researchers in ecology and evolutionary biology. We used two unpublished datasets, one from evolutionary ecology (blue tit, <em>Cyanistes caeruleus</em>, to compare sibling number and nestling growth) and one from conservation ecology (<em>Eucalyptus</em>, to compare grass cover and tree seedling recruitment). The project leaders recruited 174 analyst teams, comprising 246 analysts, to investigate the answers to prespecified research questions. Analyses
conducted by these teams yielded 141 usable effects (compatible with our meta-analyses and with all necessary information provided) for the blue tit dataset, and 85 usable effects for the <em>Eucalyptus</em> dataset.
We found substantial heterogeneity among results for both datasets, although the patterns of variation differed between them. For the blue tit analyses, the average effect was convincingly negative, with less
growth for nestlings living with more siblings, but there was near continuous variation in effect size from large negative effects to effects near zero, and even effects crossing the traditional threshold of statistical significance in the opposite direction. In contrast, the average relationship between grass cover and <em>Eucalyptus</em> seedling number was only slightly negative and not convincingly different from zero, and most effects ranged from weakly negative to weakly positive, with about a third of effects crossing the traditional threshold of significance in one direction or the other. However, there were also several striking outliers in the <em>Eucalyptus</em> dataset, with effects far from zero. For both datasets, we found substantial variation in the variable selection and random effects structures among analyses, as well as in the ratings of the analytical methods by peer reviewers, but we found no strong relationship between any of these and deviation from the meta-analytic mean. In other words, analyses with results that were far from the mean were no more or less likely to have dissimilar variable sets, use random effects in their models, or receive poor peer reviews than those analyses that found results that were close to the mean. The existence of substantial variability among analysis outcomes raises important questions about how ecologists and evolutionary biologists should interpret published results, and how they should conduct analyses in the future.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Data science</div><div class="cs-filter">Ecology</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="https://bmcbiol.biomedcentral.com/counter/pdf/10.1186/s12915-024-02101-x.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfLink" width="30" height="30" decoding="async" aria-hidden="true"></a><a href="/assets/documents/publications/s12915-024-02101-x.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a><a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-024-02101-x" target="_blank"><img src="../assets/icons/link.svg" alt="webLink" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/physiologicalresponses.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/physiologicalresponses.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/physiologicalresponses.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Frontiers</span>
                            <span class="cs-name"><p>Physiological cold tolerance evolves faster than climatic niches in plants</p>
</span>
                            <span class="cs-publication-author">Yin Wen, Qing Ye, <span class="cs-highlight-author">Cristian Román-Palacios</span>, Hui Liu, Guilin Wu</span>
                            <span><span class="cs-publication-year">2023</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Understanding how plants respond to thermal stress is central to predicting plant responses and community dynamics in natural ecosystems under projected scenarios of climate change. Although physiological tolerance is suggested to evolve slower than climatic niches, this comparison remains to be addressed in plants using a phylogenetic comparative approach. In this study, we compared i) the evolutionary rates of physiological tolerance to extreme temperatures with ii) the corresponding rates of climatic niche across three major vascular plant groups. We further accounted for the potential effects of hardening when examining the association between physiological and climatic niche rates. We found that physiological cold tolerance evolves faster than heat tolerance in all three groups. The coldest climatic-niche temperatures evolve faster than the warmest climatic-niche temperatures. Importantly, evolutionary rates of physiological cold tolerance were faster than rates of change in climatic niches. However, an inverse association between physiological cold tolerance and responding climatic niche for plants without hardening was detected. Our results indicated that plants may be sensitive to changes in warmer temperatures due to the slower evolutionary rates of heat tolerance. This pattern has deep implications for the framework that is being used to estimate climate-related extinctions over the upcoming century</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2023_Physiological cold tolerance evolves faster _Wen_etAl.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/digestunveilingevolution.svg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/digestunveilingevolution.svg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/digestunveilingevolution.svg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Evolution</span>
                            <span class="cs-name"><p>Digest: Unveiling the role of chemical communication in lizard diversification</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span></span>
                            <span><span class="cs-publication-year">2023</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Although many studies looking at the role that sexual selection plays in macroevolution have focused on the involvement of visual and acoustic signals, the potential influence of chemical communication remains unexplored. Instead of focusing on well-studied instances of sexually selected traits, Murali et al. analyze chemical communication to test whether sexual selection could help explain large-scale species diversity patterns in lizards. Their results suggest that, at least in lizards, chemical communication is uncoupled from differential diversification dynamics across clades.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2023_Digest_unveiling the role_Roman_palacios.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/thephrutarpackage.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/thephrutarpackage.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/thephrutarpackage.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Methods in Ecology and Evolution</span>
                            <span class="cs-name"><p>The phruta R package: increasing access, reproducibility, and transparency in phylogenetic analyses</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span></span>
                            <span><span class="cs-publication-year">2023</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Current practices for assembling phylogenetic trees often recur to sequence data stored in GenBank. However, the molecular and taxonomic make-up of sequences deposited in GenBank is generally not very clear. phruta, a newly developed r package, is designed to primarily improve access to genetic data stored in GenBank. Functions in phruta further enable users to assemble single- and multi-gene molecular datasets, and run basic phylogenetic tasks, all within r. The structure of the functions implemented in phruta aim to allow users to assemble simple workflows for particular tasks, which are in turn expected to increase reproducibility of relatively simple phylogenies. To support the use of phruta by researchers in different fields with variable levels of coding expertise, this paper presents the first version of salphycon, a shiny web app that aims to increase access to the fundamental functions in the phruta r package.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div><div class="cs-filter">Software</div><div class="cs-filter">Data science</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2023_The phruta R package.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a><a href="https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.14147" target="_blank"><img src="../assets/icons/link.svg" alt="webLink" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.11646/zootaxa.5188.6.3" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/goniurosauruswangshu.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/goniurosauruswangshu.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/goniurosauruswangshu.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Zootaxa</span>
                            <span class="cs-name"><p>Goniurosaurus wangshu sp. nov., a new species of Tiger Gecko from Guangdong, China (Squamata: Eublepharidae)</p>
</span>
                            <span class="cs-publication-author">Xiao-Yu Zhu, Guang-Yu Chen, <span class="cs-highlight-author">Cristian Román-Palacios</span>, Zheng Li, and Zhu-Qing He</span>
                            <span><span class="cs-publication-year">2022</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>A new species of Tiger Gecko in the genus Goniurosaurus is described from Guangdong, China. This new species, Goniurosaurus wangshu sp. nov. is the fifth species of Goniurosaurus found in karst topography of Guangdong. It is distinguished morphologically from congeners in this region by the combination of thirteen precloacal pores in male; dorsal ground color of head, body, and limbs in adults yellow; and iris orange. To validate our morphological inferences and support the description of Goniurosaurus wangshu sp. nov. as a new species, we sequenced 16S, CMOS, cytb, and RAG1 gene regions and performed molecular phylogenetic analyses. Phylogenetic analysis places the new species within the G. yingdeensis group and highlights the evolutionary distinction between this new species and other described congeners</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/ZhuEtAl_Gwangshu.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.1111/ele.13999" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/theoriginsofglobalbiodiversity.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/theoriginsofglobalbiodiversity.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/theoriginsofglobalbiodiversity.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Ecology letters</span>
                            <span class="cs-name"><p>The origins of global biodiversity on land, sea, and freshwater</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Daniela Moraga-Lopez, and John J. Wiens</span>
                            <span><span class="cs-publication-year">2022</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Many biodiversity studies focus on explaining high tropical species richness, but an equally dramatic yet understudied pattern involves the divergent richness of land, sea and freshwater. Here, we reveal the origins of these richness differences among habitats across animals and plants. Most plant and animal species are terrestrial, although these habitats cover only ~28% of Earth's surface. Marine habitats have fewer species over a larger area (~70%). Freshwater habitats have relatively high richness and exceptional phylogenetic diversity given their tiny area (2%). The relative richness of habitats is related to variation in diversification rates. Based on ancestral reconstructions of habitat, we find that most marine species are descended from marine ancestors and most terrestrial species from freshwater ancestors. Yet, most extant animal richness in freshwater is derived from terrestrial ancestors. Overall, our results reveal the origins of fundamental but neglected biodiversity patterns, and highlight the conservation importance of freshwater habitats</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Ecology</div><div class="cs-filter">Evolution</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2022_Roman-Palacios_et_al._EL_.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.11646/zootaxa.5162.4.4" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/anewspeciesofwhipspidersaraxsinesis.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/anewspeciesofwhipspidersaraxsinesis.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/anewspeciesofwhipspidersaraxsinesis.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Zootaxa</span>
                            <span class="cs-name"><p>A new species of whip spider, Sarax sinensis sp. nov., from Fujian, China (Arachnida: Amblypygi: Charinidae)</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Yherson Franchesco Molina-Henao, and Michael S. Barker. 2020.</span>
                            <span><span class="cs-publication-year">2022</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>To date, only two whip spider species have been recorded in China. We describe a new species, Sarax sinensis sp. nov., from Fujian, China. This species is morphologically similar to S. ioanniticus (Kritscher, 1959), S. israelensis (Miranda et al., 2016), and S. seychellarum (Kraepelin, 1898), but can be distinguished by the combination of the following characters: 35 segments in leg I tarsus, eight teeth on cheliceral claw, and four dorsal and ventral spines respectively on pedipalp femur. To examine the evolutionary history of S. sinensis sp. nov., we sequenced 12S, 16S, and COI gene regions of our specimens and inferred its phylogenetic position. The inferred phylogenetic trees placed the new species within Sarax, with its closest relative being distributed across the western Asia. The type specimens are deposited in the Museum of Biology, East China Normal University (ECNU).</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2022_A new species of whip spider, Sarax sinensis_WuEtAl_SS.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.7717/peerj.12450" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/treedata.tablerwrappperfordatatable.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/treedata.tablerwrappperfordatatable.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/treedata.tablerwrappperfordatatable.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">PeerJ</span>
                            <span class="cs-name"><p>treedata.table: a wrapper for data.table that enables fast manipulation of large phylogenetic trees matched to data</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, April Wright, and Josef Uyeda</span>
                            <span><span class="cs-publication-year">2021</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>The number of terminals in phylogenetic trees has significantly increased over the last decade. This trend reflects recent advances in next-generation sequencing, accessibility of public data repositories, and the increased use of phylogenies in many fields. Despite R being central to the analysis of phylogenetic data, manipulation of phylogenetic comparative datasets remains slow, complex, and poorly reproducible. Here, we describe the first R package extending the functionality and syntax of data.table to explicitly deal with phylogenetic comparative datasets. treedata.table significantly increases speed and reproducibility during the data manipulation steps involved in the phylogenetic comparative workflow in R. The latest release of treedata.table is currently available through CRAN (https://cran.r-project.org/web/packages/treedata.table/). Additional documentation can be accessed through rOpenSci (https://ropensci.github.io/treedata.table/).</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div><div class="cs-filter">Software</div><div class="cs-filter">Data science</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2021_Roman-Palacios_treedata.table.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.11646/zootaxa.5082.1.6" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/anewspeciesofwhipspiderweygoldtiahainanensis.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/anewspeciesofwhipspiderweygoldtiahainanensis.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/anewspeciesofwhipspiderweygoldtiahainanensis.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Zootaxa</span>
                            <span class="cs-name"><p>A new species of whip spider, Weygoldtia hainanensis sp. nov., from Hainan, China (Arachnida: Amblypygi: Charinidae)</p>
</span>
                            <span class="cs-publication-author">Xiao-Yu, Zhu, Shi-Yang Wu, S. Y., Yi-Juao Liu, Chris R. Reardon, <span class="cs-highlight-author">Cristian Román-Palacios</span>, Zheng Li, Zhu-Qing He</span>
                            <span><span class="cs-publication-year">2021</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>To date, only one species of whip spider has been recorded in China. Here, we describe a new species, Weygoldtia hainanensis sp. nov., from Hainan, China. The new species is morphologically similar to W. davidovi (Fage, 1946) and W. consonensis Miranda et al. 2021, but can be distinguished with a combination of the following characters: 26 segments in tibia I, 6-7 teeth on chelicerae, distitibia IV trichobothria sc and sf series each with 10-11 trichobothria. To validate our morphological inferences and support the erection of W. hainanensis sp. nov. as a new species, we sequenced the COI gene region for two individuals and performed molecular phylogenetic analyses. The inferred phylogenetic trees placed the new species within Weygoldtia and highlighted the evolutionary distinction between W. hainanensis sp. nov. and currently described whip spiders. The type specimens are deposited in the Museum of Biology, East China Normal University (ECNU).</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2021_Zhu_Weygoldtia.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.11646/zootaxa.4996.3.8" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/goniurosauruschengzheng.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/goniurosauruschengzheng.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/goniurosauruschengzheng.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Zootaxa</span>
                            <span class="cs-name"><p>Goniurosaurus chengzheng sp. nov., a new species of Leopard Gecko from Guangxi, China (Squamata: Eublepharidae)</p>
</span>
                            <span class="cs-publication-author">Xiao-Yu Zhu, Yi-Jiao, Liu, Yun, Bai, <span class="cs-highlight-author">Cristian Román-Palacios</span>, Zheng Li, Zhu-Qing He</span>
                            <span><span class="cs-publication-year">2021</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Six species of geckos in the genus Goniurosaurus have been recorded from Guangxi, China. Here we describe a new species, Goniurosaurus chengzheng sp. nov. The new species is similar to allied species from Guangxi, but unique in a combination of the following characters: (1) four body bands with three between limb insertions; (2) precloacal pores 20; (3) body color reddish- brown; (4) snout to eye distance: eye to ear distance &lt; 1. We used the mitochondrial genes 16S and cytb to confirm the distinctiveness of the species and place it within a molecular phylogeny of Goniurosaurus. The type specimens are deposited in the Museum of Biology, East China Normal University (ECNU).</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2021ZhuEtAl.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.1111/jeb.13884" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/animalchromosomecounts.jpeg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/animalchromosomecounts.jpeg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/animalchromosomecounts.jpeg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Journal of Evolutionary Biology</span>
                            <span class="cs-name"><p>Animal chromosome counts reveal similar range of chromosome numbers but with less polyploidy in animals compared to flowering plants</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Cesar A. Medina, Shing H. Zhan, Michael S. Barker</span>
                            <span><span class="cs-publication-year">2021</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Understanding the mechanisms that underlie chromosome evolution could provide insights into the processes underpinning the origin, persistence and evolutionary tempo of lineages. Here, we present the first database of chromosome counts for animals (the Animal Chromosome Count database, ACC) summarizing chromosome numbers for ~15,000 species. We found remarkable a similarity in the distribution of chromosome counts between animals and flowering plants. Nevertheless, the similarity in the distribution of chromosome numbers between animals and plants is likely to be explained by different drivers. For instance, we found that while animals and flowering plants exhibit similar frequencies of speciation-related changes in chromosome number, plant speciation is more often related to changes in ploidy. By leveraging the largest data set of chromosome counts for animals, we describe a previously undocumented pattern across the Tree of Life—animals and flowering plants show remarkably similar distributions of haploid chromosome numbers.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div><div class="cs-filter">Data science</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/RomanPalaciosetal2021.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.3389/feart.2021.641760" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/patternsofelementincorporation.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/patternsofelementincorporation.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/patternsofelementincorporation.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Frontiers in Earth Science</span>
                            <span class="cs-name"><p>Patterns of element incorporation in calcium carbonate biominerals recapitulate phylogeny for a diverse range of marine calcifiers</p>
</span>
                            <span class="cs-publication-author">Robert N. Ulrich, Maxence Guillermic, Julia Campbell, Abbas Hakim, Rachel Han, Shayleen Singh, Justin D. Stewart, <span class="cs-highlight-author">Cristian Román-Palacios</span>, Hannah M. Carroll, Ilian DeCorte, Rosaleen E. Gilmore, Whitney Doss, Aradhna Tripati, Justin B. Ries, Robert A. Eagle.</span>
                            <span><span class="cs-publication-year">2021</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Elemental ratios in biogenic marine calcium carbonates are widely used in geobiology, environmental science, and paleoenvironmental reconstructions. It is generally accepted that the elemental abundance of biogenic marine carbonates reflects a combination of the abundance of that ion in seawater, the physical properties of seawater, the mineralogy of the biomineral, and the pathways and mechanisms of biomineralization. Here we report measurements of a suite of nine elemental ratios (Li/Ca, B/Ca, Na/Ca, Mg/Ca, Zn/Ca, Sr/Ca, Cd/Ca, Ba/Ca, and U/Ca) in 18 species of benthic marine invertebrates spanning a range of biogenic carbonate polymorph mineralogies (low-Mg calcite, high-Mg calcite, aragonite, mixed mineralogy) and of phyla (including Mollusca, Echinodermata, Arthropoda, Annelida, Cnidaria, Chlorophyta, and Rhodophyta) cultured at a single temperature (25◦C) and a range of pCO2 treatments (ca. 409, 606, 903, and 2856 ppm). This dataset was used to explore various controls over elemental partitioning in biogenic marine carbonates, including species-level and biomineralizationpathway-level controls, the influence of internal pH regulation compared to external pH changes, and biocalcification responses to changes in seawater carbonate chemistry. The dataset also enables exploration of broad scale phylogenetic patterns of elemental partitioning across calcifying species, exhibiting high phylogenetic signals estimated from both uni- and multivariate analyses of the elemental ratio data (univariate: λ = 0– 0.889; multivariate: λ = 0.895–0.99). Comparing partial R 2 values returned from nonphylogenetic and phylogenetic regression analyses echo the importance of and show Frontiers in Earth Science | www.frontiersin.org 1 May 2021 | Volume 9 | Article 641760 Ulrich et al. Elemental Ratios Recapitulate Phylogeny that phylogeny explains the elemental ratio data 1.4–59 times better than mineralogy in five out of nine of the elements analyzed. Therefore, the strong associations between biomineral elemental chemistry and species relatedness suggests mechanistic controls over element incorporation rooted in the evolution of biomineralization mechanisms</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div><div class="cs-filter">Ecology</div><div class="cs-filter">Geosciences</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/UlrichEtAl2021.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.1111/brv.12698" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/specieationacrossthetreeoflife.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/specieationacrossthetreeoflife.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/specieationacrossthetreeoflife.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Biological Reviews</span>
                            <span class="cs-name"><p>Speciation across the Tree of Life</p>
</span>
                            <span class="cs-publication-author">Tania Hernández-Hernández, Elizabeth C. Miller, <span class="cs-highlight-author">Cristian Román-Palacios</span>, and John J. Wiens</span>
                            <span><span class="cs-publication-year">2021</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Much of what we know about speciation comes from detailed studies of well-known model systems. Although there have been several important syntheses on speciation, few (if any) have explicitly compared speciation among major groups across the Tree of Life. Here, we synthesize and compare what is known about key aspects of speciation across taxa, including bacteria, protists, fungi, plants, and major animal groups. We focus on three main questions. Is allopatric speciation predominant across groups? How common is ecological divergence of sister species (a requirement for ecological speciation), and on what niche axes do species diverge in each group? What are the reproductive isolating barriers in each group? Our review suggests the following patterns. (i) Based on our survey and projected species numbers, the most frequent speciation process across the Tree of Life may be co-speciation between endosymbiotic bacteria and their insect hosts. (ii) Allopatric speciation appears to be present in all major groups, and may be the most common mode in both animals and plants, based on non-overlapping ranges of sister species. (iii) Full sympatry of sister species is also widespread, and may be more common in fungi than allopatry. (iv) Full sympatry of sister species is more common in some marine animals than in terrestrial and freshwater ones. (v) Ecological divergence of sister species is widespread in all groups, including ~70% of surveyed species pairs of plants and insects. (vi) Major axes of ecological divergence involve species interactions (e.g. host-switching) and habitat divergence. (vii) Prezygotic isolation appears to be generally more widespread and important than postzygotic isolation. (viii) Rates of diversification (and presumably speciation) are strikingly different across groups, with the fastest rates in plants, and successively slower rates in animals, fungi, and protists, with the slowest rates in prokaryotes. Overall, our study represents an initial step towards understanding general patterns in speciation across all organisms.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div><div class="cs-filter">Ecology</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2021_HernandezHernandezEtAl.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.1111/geb.13253" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/evolutionarytimebestexplains.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/evolutionarytimebestexplains.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/evolutionarytimebestexplains.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Global Ecology and Biogeography</span>
                            <span class="cs-name"><p>Evolutionary time best explains the latitudinal diversity gradient of living freshwater fish diversity</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Elizabeth C. Miller and Cristian Román-Palacios</span></span>
                            <span><span class="cs-publication-year">2021</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Aim: The evolutionary causes of the latitudinal diversity gradient are debated. Hypotheses have ultimately invoked either faster rates of diversification in the tropics or more time for diversification owing to the tropical origins of higher taxa. Here, we perform the first test of the diversification rate and time hypotheses in freshwater ray-finned fishes, a group comprising nearly a quarter of all living vertebrates. Location: Global. Time period: 368–0 Ma. Major taxa studied: Extant freshwater ray-finned fishes. Methods: Using a mega-phylogeny of actinopterygian fishes and a global database of occurrence records, we estimated net diversification rates, the number of colonizations and regional colonization times of co-occurring species in freshwater drainage basins. We used generalized additive models to test whether these factors were related to latitude. We then compared the influence of diversification rates, numbers of colonizations, colonization times and surface area on species richness, and how these factors are related to each other. Results: Although both diversification rates and time were related to richness, time had greater explanatory power and was more strongly related to latitude than diversification rates. Other factors (basin surface area and number of colonizations) also helped to explain richness but were unrelated to latitude. The most diverse freshwater basins of the world (Amazon and Congo rivers) were dominated by lineages having Mesozoic origins. The temperate groups dominant today arrived near the Cretaceous–Palaeogene boundary, leaving comparatively less time to build richness. Diversification rates and colonization times were inversely related: recently colonized basins had the fastest rates, whereas ancient species-rich faunas had slower rates. Main conclusions: We concluded that time is the leading driver of latitudinal disparities in richness in freshwater fish faunas. We suggest that the most likely path to building very high species richness is through diversification over long periods of time, rather than through rapid diversification.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div><div class="cs-filter">Ecology</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2021_MillerRoman_freshwater.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.11646/zootaxa.4852.2.6" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/goniurosaurusgezhi.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/goniurosaurusgezhi.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/goniurosaurusgezhi.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Zootaxa</span>
                            <span class="cs-name"><p>Goniurosaurus gezhi, a new gecko species from Guangxi, China (Squamata: Sauria: Eublepharidae)</p>
</span>
                            <span class="cs-publication-author">Xiao-Yu Zhu, Guang-Yu Chen, <span class="cs-highlight-author">Cristian Román-Palacios</span>, Zheng Li, and Zhu-Qing He</span>
                            <span><span class="cs-publication-year">2020</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Five species of geckos in the genus Goniurosaurus had been recorded from Guangxi, China. Here we describe a new species, Goniurosaurus gezhi sp. nov. Zhu, He &amp; Li. The new species is similar to those found in Guangxi and Guizhou provinces of China and Northern Vietnam, but unique in a combination of the following characters: (1) three body bands between limb insertions; (2) precloacal pores 18–20; (3) body small (SVL=70.6–83.8 mm); (4) body color orange to yellow. We evaluated the phylogenetic position of this new species based on the 16S mitochondrial gene. Molecular phylogenies validate this new species as distinct to currently described lineages within Goniurosaurus. The type specimens are deposited in the Museum of Biology, East China Normal University (ECNU).</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/ZhuEtAl2020.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.1098/rspb.2020.0962" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/polyploidyincreasesoverall.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/polyploidyincreasesoverall.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/polyploidyincreasesoverall.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Proceedings of the Royal Society B</span>
                            <span class="cs-name"><p>Polyploidy increases overall diversity despite higher turnover than diploids in the Brassicaceae</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Yherson Franchesco Molina-Henao, and Michael S. Barker</span>
                            <span><span class="cs-publication-year">2020</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Although polyploidy is widespread across the plant Tree of Life, its long-term evolutionary significance is still poorly understood. Here, we examine the effects of polyploidy in explaining the large-scale evolutionary patterns within angiosperms by focusing on a single family exhibiting extensive interspecific variation in chromosome numbers. We inferred ploidy from haploid chromosome numbers for 80% of species in the most comprehensive species-level chronogram for the Brassicaceae. After evaluating a total of 94 phylogenetic models of diversification, we found that ploidy influences diversification rates across the Brassicaceae. We also found that despite diversifying at a similar rate to diploids, polyploids have played a significant role in driving present-day differences in species richness among clades. Overall, in addition to highlighting the complexity in the evolutionary consequences of polyploidy, our results suggest that rare successful polyploids persist while significantly contributing to the long-term evolution of clades. Our findings further indicate that polyploidy has played a major role in driving the longterm evolution of the Brassicaceae and highlight the potential of polyploidy in shaping present-day diversity patterns across the plant Tree of Life.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2020_Roman-Palaciosetal_Brassicaceae_.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.1073/pnas.1913007117" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/recentresponsestoclimatechange.jpeg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/recentresponsestoclimatechange.jpeg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/recentresponsestoclimatechange.jpeg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">PNAS</span>
                            <span class="cs-name"><p>Recent responses to climate change reveal the drivers of species extinction and survival</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, and John J. Wiens</span>
                            <span><span class="cs-publication-year">2020</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Climate change may be a major threat to biodiversity in the next 100 years. Although there has been important work on mechanisms of decline in some species, it generally remains unclear which changes in climate actually cause extinctions, and how many species will likely be lost. Here, we identify the specific changes in climate that are associated with the widespread local extinctions that have already occurred. We then use this information to predict the extent of future biodiversity loss and to identify which processes may forestall extinction. We used data from surveys of 538 plant and animal species over time, 44% of which have already had local extinctions at one or more sites. We found that locations with local extinctions had larger and faster changes in hottest yearly temperatures than those without. Surprisingly, sites with local extinctions had significantly smaller changes in mean annual temperatures, despite the widespread use of mean annual temperatures as proxies for overall climate change. Based on their past rates of dispersal, we estimate that 57–70% of these 538 species will not disperse quickly enough to avoid extinction. However, we show that niche shifts appear to be far more important for avoiding extinction than dispersal, although most studies focus only on dispersal. Specifically, considering both dispersal and niche shifts, we project that only 16–30% of these 538 species may go extinct by 2070. Overall, our results help identify the specific climatic changes that cause extinction and the processes that may help species to survive</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Climate change</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2020_Roman-Palacios_Wiens_PNAS.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.1038/d41586-019-00845-z" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/fivetipsfornailingyourusbased.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/fivetipsfornailingyourusbased.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/fivetipsfornailingyourusbased.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Nature</span>
                            <span class="cs-name"><p>Five tips for nailing your US-based graduate-school applications</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span></span>
                            <span><span class="cs-publication-year">2019</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Cristian Román-Palacios discusses how to deal with the bureaucracy and make a great case for yourself.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Academia</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2019_Nature_.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.1002/evl3.127" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/evolutionofdiet.jpeg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/evolutionofdiet.jpeg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/evolutionofdiet.jpeg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Evolution letters</span>
                            <span class="cs-name"><p>Evolution of diet across the Animal Tree of Life</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Joshua Scholl, and John J. Wiens</span>
                            <span><span class="cs-publication-year">2019</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>What an animal eats is a fundamental aspect of its biology, but the evolution of diet has not been studied across animal phylogeny. Here, we performed a large-scale phylogenetic analysis to address three unresolved questions about the evolution of animal diets. (i) Are diets conserved across animal phylogeny? (ii) Does diet influence rates of species proliferation (diversification) among animal phyla? (iii) What was the ancestral diet of animals and major animal clades? We analyzed diet data for 1087 taxa, proportionally sampled among animal phyla based on the relative species richness of phyla. Our survey suggests that across animals, carnivory is most common (63%), herbivory less common (32%), and omnivory relatively rare (3%). Despite considerable controversy over whether ecological traits are conserved or labile, we found strong conservatism in diet over extraordinarily deep timescales. We found that diet is unrelated to rates of species diversification across animal phyla, contrasting with previous studies showing that herbivory increased diversification within some important groups (e.g., crustaceans, insects, and mammals). Finally, we estimated that the ancestor of all animals was most likely carnivorous, as were many major phyla (e.g., arthropods, molluscs, and chordates). Remarkably, our results suggest that many carnivorous species living today may have maintained this diet through a continuous series of carnivorous ancestors for &gt;800 million years.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div><div class="cs-filter">Ecology</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2019_Evolution of diet across the Animal Tree of Life_Roman-Palacios_et_al_EvoLett_2019.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.1111/evo.13574" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/bammgivesmisleading.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/bammgivesmisleading.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/bammgivesmisleading.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Evolution</span>
                            <span class="cs-name"><p>BAMM gives misleading results in simulated and empirical datasets</p>
</span>
                            <span class="cs-publication-author">Andreas L. Meyer, <span class="cs-highlight-author">Cristian Román-Palacios</span>, and John J. Wiens</span>
                            <span><span class="cs-publication-year">2018</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>In a previous paper, we used simulations and empirical data to show that BAMM (Bayesian Analysis of Macroevolutionary Mixtures) can give misleading estimates of rates and rate shifts. In simulations, BAMM underestimated rate shifts across every tree analyzed, and assigned incorrect rates to most clades in most trees. In empirical analyses, BAMM behaved as expected from simulations, and assigned different rates to clades when clades were analyzed alone versus across the tree (i.e., with rate heterogeneity). Rabosky recently criticized our paper, focusing primarily on the idea that our comparison of BAMM to another approach (methodof-moments estimators of Magallon and Sanderson, or MS estimators) was unfair to BAMM. Here, we provide further evidence ´ that BAMM gives misleading rate estimates in empirical studies. We then describe how Rabosky’s rown method comparisons were either acknowledged as being problematic or were described inaccurately (to favor BAMM). Finally, we show that the MS estimators can perform well when rates vary over time, despite untested assertions that they require constant rates to be accurate. Many other methods are available for analyzing diversification rates: we argue that BAMM should be avoided for estimating both diversification rates and rate shifts.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div><div class="cs-filter">Data science</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2018_BAMM gives misleading results _Meyer_et_al_Evol_2018_BAMM.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><span class="__dimensions_badge_embed__" data-doi="10.32870/dugesiana.v25i2.7031 " data-style="small_circle"></span></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/checklistofpsocopterapsocodea.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/checklistofpsocopterapsocodea.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/checklistofpsocopterapsocodea.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Dugesiana</span>
                            <span class="cs-name"><p>Checklist of ‘Psocoptera’(Psocodea) of Colombia and identification key to families</p>
</span>
                            <span class="cs-publication-author">Alfonso N. Garcia-Aldrete, Nancy S. Carrejo, Julian Mendivil, Nadia Calderón, Oscar Saenz, Jefferson Panche, <span class="cs-highlight-author">Cristian Román-Palacios</span>, and Ranulfo Obando-Obando</span>
                            <span><span class="cs-publication-year">2018</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>We present a list of 720 species of ‘Psocoptera’ recorded in Colombia, based on the examination of specimens deposited in the collections of the Alexander von Humboldt Institute and in the Entomological Museum of the Universidad del Valle. We include records of specimens recently collected in several National Natural Parks and other natural areas in Colombia. Species previously recorded in the literature are also listed here. The number of psocids recorded in Colombia until 2002 (79 species) corresponded to only 11.0% of what is currently known based on our study. We highlight that about 60% of the species recorded since 2010 were undescribed, 135 of which have been recently described by our research group. An identification key to the 28 psocid families known in Colombia is included. Even though psocids have been largely ignored from the taxonomic research in Colombia, we show that the group diversity is still to be uncovered in the country. Further studies must be conducted to understand the distribution of psocids among and within localities, and emphasize that additional extensive fieldwork is highly required.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2018_Checklist of ‘Psocoptera’(Psocodea)_Garcia-Aldrete et al 2018_Psocoptera_Colombia.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.1111/jbi.13366" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/thetortoiseandthefinch.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/thetortoiseandthefinch.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/thetortoiseandthefinch.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Journal of Biogeography</span>
                            <span class="cs-name"><p>The Tortoise and the Finch: testing for island effects on diversification using two iconic Galápagos radiations</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, and John J. Wiens</span>
                            <span><span class="cs-publication-year">2018</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Aims: Islands are widely recognized as natural laboratories for evolutionary studies, but many questions about evolution on islands remain unresolved. Here we address two general questions from a macroevolutionary perspective. First, do lineages on islands have increased diversification rates relative to mainland lineages? Second, does the same geographical context (e.g., same archipelago) have similar effects on diversification in unrelated groups? We focused on Darwin’s finches and Galapagos tortoises, two endemic radiations from the Galapagos Islands, and the larger families in which they are embedded. Location: Global. Methods: We estimated a new time-calibrated phylogeny for tortoises (Testudinidae). Then, we examined their macroevolutionary patterns and compared them to those of Darwin’s finches and relatives (Thraupidae), using a published thraupid tree. Specifically, we estimated and compared diversification rates between islands and the mainland and between the Galapagos Islands and all other regions, using clade-based and species-based approaches. Results: Contrary to expectations, occurrence on islands in general did not significantly increase diversification rates in tanagers or tortoises. However, occurrence in the Galapagos Islands in particular was associated with increased speciation and diversification rates, and explained ~28% of the variation in diversification rates for thraupids and ~46% for testudinids. Both Darwin’s finches and Galapagos tortoises were unique within each family in exhibiting the highest diversification rates. The congruence of these macroevolutionary patterns between both radiations supports a strong “place-dependent” effect on diversification associated with the Galapagos. Finally, we found that Darwin’s finches diversified ~2–8 times faster than Galapagos tortoises. Main conclusions: Our results show that occurring on islands in general did not increase diversification rates in these clades, but occurrence in the Galapagos did. We also show that dramatic local-scale differences in diversification rates between clades in the Galapagos parallel global-scale differences in diversification rates between these families and between birds and turtles overall.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2018_The Tortoise and the Finch _Roman_Palacios_Wiens_JBI_2018.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.1016/j.ympev.2018.06.012" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/whendidanolislizardsdiverge.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/whendidanolislizardsdiverge.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/whendidanolislizardsdiverge.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Molecular Phylogenetics and Evolution</span>
                            <span class="cs-name"><p>When did Anolis lizards diverge? A comparison of multiple dating strategies</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Rosario Castañeda, and Jose Julian Tavera</span>
                            <span><span class="cs-publication-year">2018</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Whereas most of the studies that discuss the evolutionary divergence of Anolis lizards have dated the clade’s crown group in between 31 and 64Ma, a single study has recovered a significantly older age for the same node (87Ma). These differences also entail notable consequences on the preferred biogeographical hypothesis for the whole clade. Here we analyze a total of seven dating strategies by combining three calibration sources in independent BEAST runs to infer the most probable divergence timing for anole lizards (a mitochondrial rate for ND2 gene, the Anolis dominicanus fossil, and a group of fossils assigned to the Priscagamines, Iguanines, and Idontosaurus clades). Based on the estimated timing, we also addressed whether chronograms differ the most in deeper or shallower nodes by exploring the trend in the standard deviation of mean ages between chronograms across time. Next, we focus on the pattern for a single shallow node by hypothesizing the biogeography of the island-endemic Malpelo anole (Anolis agassizi), and evaluating the temporal congruence between the species’ divergence and the island geology. The estimated set of ages suggests that anoles most likely diverged 72Ma (71–73Ma), with the crown group established around 58Ma (51–65Ma). Dispersal is therefore supported as the major driver in the biogeography of the group (and in Caribbean lineages in particular). Our analyses also indicated that (1) rate-based analyses pulled dates toward younger ages, (2) the differences in node ages between chronograms decrease towards the tips regardless of the position of the constrained node, and that (3) the estimated age for deep nodes (e.g. Anolis stem) is highly influenced when deep nodes are also constrained. The latter two results imply that the estimated age for shallower nodes is largely unaffected by the used temporal constraint. The congruence of all chronograms for the Malpelo anole also support this finding. Anolis agassizi was found to have diverged before the emergence of Malpelo island in each analysis (anole: 19–31Ma vs. Malpelo island: 16–17Ma). We recommend when performing absolute dating analyses to first test for sequence saturation in the analyzed dataset (especially when calibrations are based on molecular rates). Our study also points out the importance of using of multiple node constraints, especially when placed deeply in the tree, for fossil–based divergence dating analyses.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2018_roman-palacios_et_al_anolis.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.1163/1876312x-00002197" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/molecularseaskaters.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/molecularseaskaters.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/molecularseaskaters.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Insect Systematics and Evolution</span>
                            <span class="cs-name"><p>Molecular phylogeny of sea skaters and their freshwater relatives (Hemiptera: Heteroptera: Gerridae: Halobatinae)</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Jakob Damgaard, Lanna Cheng, Felipe Ferraz Figueiredo Moreirae, Tom A. Weir, and Herbert Zettle</span>
                            <span><span class="cs-publication-year">2018</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>We investigated the phylogenetic relationships among seven of the ten Halobatinae genera (Heteroptera: Gerridae) based on COI+II, 16S rRNA, and 28S rRNA genes. Our analyses recovered monophyly of Halobatinae, and suggested paraphyly of Metrocorini caused by the position of Ventidius and Esakia. Since our phylogenies did not infer monophyly of the subgenus Halobates (s.str.) within Halobatini, we synonymized Austrobates and Halobates (Hilliella) with Halobates. We confirmed that (1) the limnic lifestyle of Metrocorini was ancestral in Halobatinae, (2) the marine lifestyle evolved only once in the common ancestor of Asclepios + Halobates, (3) the limnic lifestyle of some members of Halobates was independently derived from marine ancestors, and that (4) the open ocean was colonized at least three times in Halobates. A catalogue of Halobatinae organized according to an updated classification is presented, including all known geographic distributions, bibliographical references, and additional notes to all species of the subfamily.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2018_Molecular phylogeny of sea skaters_Roman-Palacios_et_al._Halobatinae.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.15517/rbt.v66i3.30818" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/onthedatadeficientamphibians.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/onthedatadeficientamphibians.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/onthedatadeficientamphibians.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Revista de Biologia Tropical</span>
                            <span class="cs-name"><p>On the Data Deficient amphibians from Colombia</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, and Alejandro Valencia-Zuleta</span>
                            <span><span class="cs-publication-year">2018</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Whereas more than 10 % of global amphibian richness is known to occur in Colombia, almost 16 % of these species are currently classified as Data Deficient according to the IUCN. These estimates suggest that the available data for a large portion of amphibians occurring in Colombia is insufficient to assess extinction risk. Here we aim to (1) review the available information on the distribution of the Colombian Data Deficient (DD hereafter) amphibians, (2) analyze their geographic distribution, and (3) evaluate the relationship between anthropogenic impact and their current conservation status. For this, we first compiled geographical records for the DD amphibian species using primary sources. Geographical records were obtained mainly from taxonomic descriptions and non-systematic surveys. We then estimated the geographical range and inferred the potential distribution for each species using letsR and MaxEnt, respectively. We quantified the human footprint for each species and tested the relationship between spatial distribution and anthropogenic change across populations. Analyses are here based on 128 of the 129 DD amphibian species that occur in Colombia. We found that most of these species were recently described and have small geographic ranges. A large proportion of these DD amphibians inhabit the Colombian Andes, and their populations have been strongly affected by human activities. Overall, the spatial clustering suggests that many of these species have faced similar environmental and anthropogenic pressures that have contributed to their rareness. We also suggest that the conservation status for several of the analyzed DD amphibians should be changed to account for the threats they face. Rev. Biol. Trop. 66(3): 1272-1281. Epub 2018 September 01.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Ecology</div><div class="cs-filter">Data science</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2018_On the Data Deficient amphibians _Roman-Palacios_&amp;_Valencia-Zuleta 2018_Colombian_DD.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.15517/rbt.v66i3.30818" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/onthedivergenceofpsocidae.png">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/onthedivergenceofpsocidae.png">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/onthedivergenceofpsocidae.png" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Boletin del Museo de Entomologia de la Universidad del Valle</span>
                            <span class="cs-name"><p>On the divergence of Psocidae (Psocodea, Psocomorpha): Fossils or biogeography</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Yherson Franchesco Molina-Henao, and Ranulfo Gonzalez-Obando</span>
                            <span><span class="cs-publication-year">2017</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>With more than 900 valid species, Psocidae is the most diverse family of psocodeans. Even though the availability of molecular has allowed hypothesizing on the phylogenetic relations for the different lineages, the evolutionary timing is still poorly discussed and the influence of intrinsic and extrinsic factors on their diversification is hitherto unknown. In this paper, we aim to test the temporal congruence of three evidences (i.e., two fossils and one biogeographical constraint), and discuss the correlation between clade’s age and other major events (e.g., geological, ecological). We found that family Psocidae likely diverged during the Mesozoic – Paleozoic transition. Moreover, the subsequent splits of the major sub-lineages occurred in a short time interval between 68 to 77 Mya. These dates also correspond to diversification shifts in multiple angiosperm lineages and the increase in global temperature.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Evolution</div><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2017_On the divergence of Psocidae_Roman-Palacios et al. 2017_Psocidae_Divergence.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><span class="__dimensions_badge_embed__" data-doi="10.21068/c2017.v18n01a16" data-style="small_circle"></span></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/listaanotada.png">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/listaanotada.png">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/listaanotada.png" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Biota Colombiana</span>
                            <span class="cs-name"><p>Lista anotada de la herpetofauna del departamento del Quindío, Colombia</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Sara Fernández–Garzón, Alejandro Valencia–Zuleta, Andrés F. Jaramillo–Martínez, and Ronald A. Viáfara–Vega</span>
                            <span><span class="cs-publication-year">2017</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>The increasing uncertainty about the conservation status of the different biological groups emphasizes the importance of concentrating efforts on gathering diversity records. In Colombia, biological explorations have revealed priority areas for conservation, but the lack of information about topics as species distribution, restricts both the emphasis and efficiency of conservation efforts. In particular, Quindío department represents an important area of faunal interest due to its geographical position and climatic conditions. However, still lacks of basic information for an comprehensive discussion of local diversity. This works aims to summarize the information on richness, geographic distribution and diversity of the herpetofauna in the department. Published information from different bibliographic resources indicates that about 108 species of amphibians and reptiles (48 and 57 spp. respectively) coexist within the political boundaries. Among these, the genus Anolis and Pristimantis recorded the highest species diversity. Despite the reduced department’s size, the species that have been recorded exhibit remarkable ecological importance and also act as a reservoir of diversity regarding surrounding areas.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2017_Lista anotada de la herpetofauna_Roman-Palacios et al. 2017_Quindio_Herps.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.17151/bccm.2016.20.2.14" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/usodemicrohabitat.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/usodemicrohabitat.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/usodemicrohabitat.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Boletin Cientifico Centro de Museos Museo de Historia Natural de la Universidad de Caldas</span>
                            <span class="cs-name"><p>Uso de microhábitat por anuros en un fragmento de bosque seco intervenido del Magdalena medio, Guarinocito, Colombia</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Sara Fernández-Garzon, Jenny Johana Gallo-Franco, Alan Giraldo-Lopez, Monica Hernandez-Lopez, and Wilmar Bolivar-Garcia</span>
                            <span><span class="cs-publication-year">2017</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>The anuran assemblages in three habitats (forest, lake and quarry) with different degrees of human disturbance present in a dry forest fragment in the Farm La Española, Guarinocito, Caldas, were compared. Extensive routes samplings were conducted using visual encounter surveys (VES), from which abundance and microhabitat use were recorded for each one of the species within the different habitats, and the record of environmental variables describing the sampling area such as temperature, relative humidity, and water ponds perimeter, was carried out. A total of 13 anurans species were found, among which Engystomops pustulosus recorded the most relative abundance with a catch rate of 5.16 individuals per hour. Significant differences were found in diversity between the studied habitats being lower in the forest where there were lower richness but better dominance. The rate of replacement of species between habitats was 71%, showing pond and quarry the greater similarity in anurofauna composition. The use of substrate by each species was different between habitats (χ2, p&lt;0.05) and only two species (Leptodactylus fragilis and L. colombiensis) showed high variability in the modified habitats.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Ecology</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2016_Uso de microhábitat por anuros _Roman-Palacios et al_2016_Guarinocito.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><span class="__dimensions_badge_embed__" data-doi="10.5852/ejt.2016.194" data-style="small_circle"></span></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/dietayreproduccion.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/dietayreproduccion.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/dietayreproduccion.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Revista MVZ Cordoba</span>
                            <span class="cs-name"><p>Dieta y reproducción de <em>Bryconamericus caucanus</em> (Characiformes: Characidae) en La Venada, Río Quindío, Colombia</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, and Cesar Román Valencia</span>
                            <span><span class="cs-publication-year">2016</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Objectives: The reproductive and trophic ecology of Bryconamericus caucanus was analyzed in la Venada creek, Quindío river, Alto Cauca, Colombia. Materials and methods. Individuals were collected between January and December 2013. Multivariate analyses were performed to compare the trophic characteristics of the species between climatic periods (wet and dry), sexes, and sexual maturity (juvenile and adults). We also analyzed the condition factor (K) and the numerical (%N), volumetric (%V), and frequency (%Fo) percentages for each prey. We tested for deviations in the sex proportion using chi-squared tests and we also estimated the Gonodosomatic Index (GSI) and absolute fecundity (Fa). Results. A total of 162 individuals (SL: 40-75 mm) were here analyzed. B. caucanus is an insectivorous fish (86.47% IRI) that mainly feeds on the orders Ephemeroptera (Baetidae; 27.71%), Hymenoptera (Formicidae; 23.57%), Diptera (Simuliidae 17.36%; Chironomidae 9.79%), and Trichoptera (Hydropsychidae; 8.04). Multivariate analyses showed differences between climatic periods, with the lowest richness occurring during dry season. Between sexes, females showed the widest trophic niche. Adults, on the other hand, showed a wider trophic niche than juveniles. The species has two reproductive peaks during rainy seasons which are also related to the increase in the trophic richness during the same periods (March-April, September-November). Conclusions. We found that B. caucanus has an incipient trophic differentiation between ontogenic stages and sexes. Trophic differences are remarkable between climatic periods. The two spawning periods for the species occur during the beginning of the wet seasons (March-April, September-November).</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Ecology</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2016_Dieta y reproducción de Bryconamericus_Roman-Palacios et al_2016_caucanus.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><span class="__dimensions_badge_embed__" data-doi="10.11646/zootaxa.4162.2.11" data-style="small_circle"></span></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/historicalbiogeographyofpsocids.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/historicalbiogeographyofpsocids.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/historicalbiogeographyofpsocids.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">European Journal of Taxonomy</span>
                            <span class="cs-name"><p>Historical biogeography of Thyrsophorini psocids and description of a new Neotropical species of Thyrsopsocopsis (Psocodea: Psocomorpha: Psocidae)</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Alfonso Neri García-Aldrete, and Ranulfo Gonzalez Obando</span>
                            <span><span class="cs-publication-year">2016</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>When based on phylogenetic proposals, biogeographic historic narratives have a great interest for hypothesizing paths of origin of the current biodiversity. Among the many questions that remain unsolved about psocids, the distribution of Thyrsophorini represents still a remarkable enigma. This tribe had been considered as exclusively Neotropical, until the description of Thyrsopsocopsis thorntoni Mockford, 2004, from Vietnam. Three hypotheses have been proposed to explain this atypical distribution, recurring to dispersal, vicariance and morphological parallelism between lineages, but the lack of evidence has not allowed a unique support. Here, we describe a new Neotropical species of Thyrsopsocopsis, and also attempt to test the three biogeographical hypotheses in a phylogenetic context. In our analyses, the position of Cycetes (Cycetini) and Thyrsopsocopsis among Thyrsophorini psocids is well resolved, suggesting a dispersal event from the Neotropics for both genera.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2016_Historical biogeography of Thyrsophorini _Roman-Palacios et al 2016_Thyrso.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><span class="__dimensions_badge_embed__" data-doi="10.11646/zootaxa.4162.2.11" data-style="small_circle"></span></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/uncoveringthediversity.png">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/uncoveringthediversity.png">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/uncoveringthediversity.png" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Zootaxa</span>
                            <span class="cs-name"><p>Uncovering the diversity of the neotropical genus Elaphopsocus (‘Psocoptera’: Psocidae: Amphigerontiinae): from one to ten species</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Ranulfo Obando-Gonzalez, and Alfonso Neri Aldrete</span>
                            <span><span class="cs-publication-year">2016</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>For 75 years, the genus Elaphopsocus was considered a monotypic lineage of neotropical psocids. As a result of recent work in South America, this genus presently includes seven species from Brazil and Colombia. We here describe three new species from the East Colombian high Andes.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2015_Uncovering the diversity of the neotropical genus Elaphopsocus _Roman-Palacios_et_al_2015_Elaphosocus.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><span class="__dimensions_badge_embed__" data-doi="	10.11646/zootaxa.4000.3.6" data-style="small_circle"></span></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/anewneotropicalgenusofblastini.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/anewneotropicalgenusofblastini.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/anewneotropicalgenusofblastini.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Zootaxa</span>
                            <span class="cs-name"><p>A new Neotropical genus of Blastini (Psocodea:’Psocoptera’: Psocidae: Amphigerontiinae)</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, and Alfonso N. García Aldrete</span>
                            <span><span class="cs-publication-year">2015</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>A monotypic genus of Psocidae (Amphigerontiinae: Blastini) from Santiago de Cali, Colombia, is here described and illustrated. It differs from Chaetopsocidus Badonnel, from the Páramo de Monserrate, near Bogotá, in having setae on the forewing veins.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2015_A new Neotropical genus of Blastini_Garcia-Aldrete_Roman-Palacios_2015_Blastini.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><div data-badge-type="donut" data-doi="10.21897/rmvz.1134" data-hide-no-mentions="true" class="altmetric-embed"></div></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/informationcriteria.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/informationcriteria.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/informationcriteria.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Comunicaciones en Estadistica</span>
                            <span class="cs-name"><p>Information criteria on multimodel selection of parametric regression: Biological applications</p>
</span>
                            <span class="cs-publication-author">Daniela Moraga Lopez, <span class="cs-highlight-author">and Cristian Román-Palacios</span></span>
                            <span><span class="cs-publication-year">2015</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>When carrying out modelling analysis using experimental data is important to obtain a measure of the relative fit of the each model as a primary selection criterion. In this sense, there are few studies based on multi-model selection techniques for the theoretical representation of data sets, so it is common to incur in a misinterpretation of the existing patterns, or even more, the incorrect extrapolation and prediction based on the wrong model. This paper is intended to evaluate in 40 datasets from various ecological published studies the effectiveness of the linear regression model designated by the authors by contrasting with six regression models using the Akaike and Bayesian information criteria, and furthermore to discuss its implications on subsequent interpretations made. It was found that the linear regression model was successful only in 13.35% of the datasets (15% of datasets for AIC and 11.7% of datasets for BIC ), but in the other hand, the logarithmic model was the most successful model in 38.5% of the cases (35% of datasets for AIC and 41.1% of the datasets for BIC), casting doubts on the efficiency of the linear regression over other types of regression under biological data.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Data science</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2015_Information criteria on multimodel selection_Moraga_Lopez_Roman-Palacios_2015_Multimodel.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><span class="__dimensions_badge_embed__" data-doi="10.11646/zootaxa.3957.2.1" data-style="small_circle"></span></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/thegenuselaphopsocusroesler.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/thegenuselaphopsocusroesler.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/thegenuselaphopsocusroesler.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Zootaxa</span>
                            <span class="cs-name"><p>The genus Elaphopsocus Roesler (Psocodea:’Psocoptera’: Psocidae) with six new species from Brazil and Colombia</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Ranulfo Gonzalez Obando, and Alfonso Neri Garcia Aldrete</span>
                            <span><span class="cs-publication-year">2015</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Six new species of Elaphopsocus Roesler from Brazil and Colombia are described and illustrated. Revised generic diagnoses are provided for Elaphopsocoides and Elaphopsocus; based on the structure of the male hypandrium and phallosome three species groups are recognized in Elaphopsocus.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2015_The genus Elaphopsocus Roesler_The_genus_Elaphopsocus.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><span class="__dimensions_badge_embed__" data-doi="10.1016/j.rmb.2015.06.009" data-style="small_circle"></span></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/habitostroficosdedosespecies.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/habitostroficosdedosespecies.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/habitostroficosdedosespecies.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Revista Mexicana de Biodiversidad</span>
                            <span class="cs-name"><p>Hábitos tróficos de dos especies sintópicas de carácidos en una quebrada de alta montaña en los Andes de Colombia</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, and Cesar Román Valencia</span>
                            <span><span class="cs-publication-year">2015</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>To analyze the degree of trophic overlap, 2 species of Neotropical characins were considered: Hemibrycon brevispini and Bryconamericus caucanus. Both species have been identified as having an insectivorous diet with a tendency to consume mayflies, hymenopterans, and dipterans. It was found that these fish share 65% of food items, which is reflected in their high trophic overlap (QS = 0.620). There is also no difference between the abundances of prey consumed. The values of trophic breadth, diversity, evenness, and richness were higher in B. caucanus than in H. brevispini, which explains the ecological flexibility exhibited by B. caucanus throughout its wide range of geographical distribution compared to H. brevispini. In these taxa, syntopy is mainly due to the high abundance of resources available during the year</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Ecology</div><div class="cs-filter">Data science</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2015_Hábitos tróficos de dos especies sintópicas_Roman-Palacios_&amp;_Roman-Valencia_2015_Two_freshwater.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><span class="__dimensions_badge_embed__" data-doi="	10.11646/zootaxa.3873.1.8" data-style="small_circle"></span></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/elaphopsicoides.png">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/elaphopsicoides.png">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/elaphopsicoides.png" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Zootaxa</span>
                            <span class="cs-name"><p>Elaphopsocoides, a new genus of Psocidae (Psocodea: ’Psocoptera’) from Valle del Cauca, Colombia</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Alfonso Neri García Aldrete, and Ranulfo Gonzalez-Obando</span>
                            <span><span class="cs-publication-year">2014</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Two new species of <em>Elaphopsocoides</em> n. gen. from Valle del Cauca, Colombia, are here described and illustrated. The new genus is related to Elaphopsocus Roesler, but differs from it mostly in the hypandrial projections and in phallosome structure. The female subgenital plate has a distinct, median, posterior projection. The types are deposited in the Entomology Museum, Universidad del Valle (MUSENUV), Santiago de Cali, Colombia.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Taxonomy</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2014_Elaphopsocoides, a new genus of Psocidae _Roman-P_et_al_2014_Elaphopsocus.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><div class="cs-badge"><span class="__dimensions_badge_embed__" data-doi="10.15446/caldasia.v36n2.47487" data-style="small_circle"></span></div><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/trophicandreproductiveecology.jpg">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/trophicandreproductiveecology.jpg">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/trophicandreproductiveecology.jpg" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Caldasia</span>
                            <span class="cs-name"><p>Trophic and reproductive ecology of a neotropical characid fish Hemibrycon brevispini (Teleostei: Characiformes)</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Cristian Román-Palacios</span>, Cesar Román Valencia, and Donald C. Taphorn</span>
                            <span><span class="cs-publication-year">2014</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Hemibrycon brevispini is a Neotropical characid fish endemic in La Venada Creek, a headwater tributary of the Quindío River of Colombia (Cauca River drainage). It is mainly a diurnal insectivore with a diet dominated by benthic dipterans (Chironomidae, Simuliidae, Psychodidae, Culicidae, Calliphoridae, Dixidae and Muscidae), hymenopterans (Formicidae and Vespidae) and ephemeropterans (Baetidae), as well as allochthonous prey and items eaten accidentally. Microhabitats of mountain streams with lower water velocity tend to have more riparian vegetation and the associated terrestrial arthropods that are consumed by H. brevispini. It has three peaks in reproduction: December, April and August. Average fecundity was 776 mature oocytes per female.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Ecology</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/2015_Trophic and reproductive ecology _Roman-Palacios_et_al_2015_Brevi.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/screenshot-2025-01-03-at-12.02.25-pm.png">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/screenshot-2025-01-03-at-12.02.25-pm.png">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/screenshot-2025-01-03-at-12.02.25-pm.png" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">bioRxiv</span>
                            <span class="cs-name"><p>SSARP: An R Package for Easily Creating Species- and Speciation- Area Relationships Using Web Databases</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Kristen Martinet</span>, <span class="cs-highlight-author">Cristian Román-Palacios</span>, Luke Harmon</span>
                            <span><span class="cs-publication-month">Jan</span>
                                    <span> </span><span class="cs-publication-year">2025</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>A universal method of quantifying patterns of biodiversity on islands is the species-area relationship (SAR). SARs visualize the relationship between species richness (the number of species) and the area of the land mass on which they live. An extension of this visualization, the speciation-area relationship (SpAR), helps researchers determine trends in speciation rate over a set of land masses. Comparing these relationships across island systems globally is an extremely difficult task because gathering and processing a large amount of species occurrence data and island data often requires researchers to conduct lengthy literature searches and combine datasets from several different sources. Here we present SSARP (Species/Speciation-Area Relationship Projector), an R package that provides a simple workflow for creating SARs and SpARs. The SSARP workflow allows users to gather occurrence data from GBIF, use mapping
tools to determine whether the GPS points in the occurrence data refer to valid land masses, associate those land masses with their areas using a built-in dataset of island names and areas, and create SARs using linear and segmented regression. SSARP also provides multiple functions for estimating speciation rates for use in creating a SpAR. Using SSARP allows researchers to dramatically increase the scope of their biodiversity research through the creation of SARs and SpARs with data from island systems across the globe.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Software</div><div class="cs-filter">Evolution</div><div class="cs-filter">Ecology</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="https://www.biorxiv.org/content/10.1101/2024.12.31.630948v1" target="_blank"><img src="../assets/icons/link.svg" alt="webLink" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div><div class="cs-item"><picture class="cs-picture">
                            <!--Mobile Image-->
                            <source media="(max-width: 600px)" srcset="/assets/images/publications/screenshot-2024-05-09-at-8.37.17-am.png">
                            <!--Tablet and up Image-->
                            <source media="(min-width: 601px)" srcset="/assets/images/publications/screenshot-2024-05-09-at-8.37.17-am.png">
                            <img loading="lazy" decoding="async" src="/assets/images/publications/screenshot-2024-05-09-at-8.37.17-am.png" alt="" width="522" height="370" aria-hidden="true">
                        </picture>
                        <div class="cs-flex-group">
                            <div aria-hidden="true" class="line line-full card-image-line"></div>
                            <span class="cs-publication-type">Remote Sensing of Environment</span>
                            <span class="cs-name"><p>Using CloudSat to advance the global precipitation climatology project (GPCP) over Antarctica</p>
</span>
                            <span class="cs-publication-author"><span class="cs-highlight-author">Mohammad Reza Ehsani</span>, Ali Behrangi, <span class="cs-highlight-author">Cristian Román-Palacios</span>, George J. Huffman, Robert F. Adler</span>
                            <span><span class="cs-publication-month">May</span>
                                    <span> </span><span class="cs-publication-year">2024</span>
                            </span>
                            </span>
                            <span class="cs-desc">
                                <div class="read-more-text"><p>Remote sensing-based precipitation products face several challenges in high latitudes and specifically over frozen surfaces (i.e., snow and ice). Consequently, precipitation estimates tend to be lower in quality over these regions, including Antarctica, the coldest continent on Earth. In this study, we developed a method for adjusting precipitation estimates over Antarctica by leveraging CloudSat's ability to capture snowfall compared to other satellite products over snow and ice surfaces. We addressed limitations of CloudSat, such as poor spatiotemporal sampling, noise, and incomplete coverage near the poles. We utilized the European Centre for Medium-Range Weather Forecasts (ECMWF) Reanalysis 5th Generation (ERA5) to guide development, particularly during the period when the TIROS Operational Vertical Sounder (TOVS) was contributing. We adjusted monthly and seasonal TOVS and Atmospheric Infrared Sounder – Infrared (AIRS-IR) precipitation biases over Antarctica at the pixel level, as these two products are the primary inputs for the Global Precipitation Climatology Project (GPCP) in high latitudes. We assessed the adjusted TOVS and AIRS-IR through analyses of geographical maps and time series of monthly and seasonal mean precipitation rates. The results are encouraging, indicating that the proposed approach could replace the current approach used in the GPCP for adjusting precipitation estimates from AIRS-IR and TOVS over Antarctica. Adjusting precipitation estimates from TOVS and AIRS-IR using the proposed approach improves the Kling-Gupta efficiency (KGE) over their entire period by 162% and 147%, respectively. Moreover, the proposed approach can be applied to adjust other precipitation products over Antarctica at the pixel level, including satellites and reanalysis products.</p>
</div>
                                <span class="read-more-btn">...read more</span>
                            </span>
                            <div aria-hidden="true" class="line line-full"></div>
                            <div class="cs-item-filters"><div class="cs-filter">Data science</div><div class="cs-filter">Climatology</div><div class="cs-filter">Remote sensing</div></div><div aria-hidden="true" class="line line-full"></div>
                                <div class="cs-item-social"><a href="/assets/documents/publications/ehsanietal2024.pdf" target="_blank"><img src="../assets/icons/pdf.svg" alt="pdfDocument" width="30" height="30" decoding="async" aria-hidden="true"></a><a href="https://www.sciencedirect.com/science/article/pii/S0034425724002177" target="_blank"><img src="../assets/icons/link.svg" alt="webLink" width="30" height="30" decoding="async" aria-hidden="true"></a></div></div>
                    </div></div>
        </div>

    </div>
</section>

<section id="copyright-notice">
    <div class="cs-container">
        <div class="cs-content">
            <h5 class="cs-title">Statement: Copyright Notice</h5>
            <p class="cs-text">
                These documents are protected by various copyright laws. However, I may distribute copies to individuals for personal, research use. Your click on any of the PDF links above constitutes your request for a personal copy of the linked article and my delivery of a personal copy. Any other use or distribution is prohibited. These are provided as a means to ensure timely dissemination of scholarly and technical work on a noncommercial basis.
            </p>
        </div>
    </div>
</section></main>
        <!-- ============================================ -->
<!--                   Footer                     -->
<!-- ============================================ -->

<footer id="footer">
    <div aria-hidden="true" class="line footer"></div>
    <div class="container">
        <div class="full-section">
            <a class="logo" href="/">
                
                    
                        <img class="cs-dark" src="../../assets/images/logos/ddl-logo.png" alt="logo" width="210" height="29" aria-hidden="true" decoding="async" loading="lazy">
                    
                
                    
                
            </a>
            <div class="credit">
                <span>Designed and hand coded by</span>
                <a href="https://columbiavalleywebdesigns.com/" target="_blank" rel="noopener">columbia valley web designs</a>
            </div>
        </div>
    </div>
</footer>

        <script>
            const appHeight = () => {
                const doc = document.documentElement
                doc.style.setProperty('--app-height', `${window.innerHeight}px`)
            }
            window.addEventListener('onload', appHeight);

            appHeight();

            const scroll = document.documentElement.scrollTop;
            if (scroll >= 50) {
                document.querySelector('body').classList.add('scroll')
            } else {
                document.querySelector('body').classList.remove('scroll')
            }
        </script>

        <script defer src="/assets/js/nav.js"></script>

        
            <script defer src="/assets//js/publications.js"></script>
        

        
    </body>
</html>
